dracut/30coreos-installer/module-setup.sh

```diff
@@ -35,6 +35,15 @@ install() {
     inst_multiple /usr/sbin/blockdev
     inst_multiple /usr/sbin/wipefs
 
+    if [ "$arch" = "s390x" ]; then
```

Comment:
Where is `$arch` being set for this, is it a dracut primitive?

Reply:
Yes, `$arch` is inherited from dracut scripts. It is widely used in dracut itself.

Reply:
Resolving.